be/src/util/callback_closure.h

```diff
@@ -34,25 +34,27 @@ namespace starrocks {
 template <typename T, typename C = void>
 class CallBackClosure : public google::protobuf::Closure {
 public:
-    CallBackClosure() : _refs(0) {}
+    CallBackClosure(const C& ctx) : _ctx(ctx) {}
     ~CallBackClosure() override = default;
 
+    bool is_idle() { return _is_idle; }
```

Comment:
Why change the `_refs` to `_is_idle`?  I think the `_refs` is more general than `_is_idle`.

Reply:
Closure is exclusive, ref can only be 0 or 1, so I replace it with bool

Reply:
fixed